lib/Repositories/ExpoFileDriver.php

```diff
@@ -13,22 +14,22 @@ class ExpoFileDriver implements ExpoRepository
      */
     private $storage = __DIR__.'/../../storage/tokens.json';
 
+    public function __construct()
+    {
+        $this->setCustomStorage();
```

Comment:
I'd also allow to setCustomStorage via `$path` param injected via constructor so that if I passed it, it would have the highest priority over the value from `.env` so that the method signature would be `setCustomStorage(string $path = null)`, wdyt?